Source/DevAddin/addinLoad.jsl

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright © 2019, SAS Institute Inc., Cary, NC, USA.  All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+Names Default To Here(0);
+// Section: Globals
+
+ut addin configuration error = Expr(
+	New Window("Error", <<Modal, Text Box("JSL Hamcrest Dev is not configured properly"));
+	Stop();
+);
+
+ut addin scratch = Convert File Path("$ADDIN_HOME(com.jmp.jslhamcrestdev)");
```

Comment:
After discussing, we are going to leave this as `Convert File Path` and assume the addin is installed. This entire file is then going to get run before every action instead of just once when JMP starts. This prevents problems when symbols are deleted.